test/extensions/quic_listeners/quiche/envoy_quic_server_stream_test.cc

```diff
@@ -151,6 +154,7 @@ INSTANTIATE_TEST_SUITE_P(EnvoyQuicServerStreamTests, EnvoyQuicServerStreamTest,
                          testing::ValuesIn({true, false}));
 
 TEST_P(EnvoyQuicServerStreamTest, GetRequestAndResponse) {
+  quic::SetVerbosityLogThreshold(1);
```

Comment:
Was this for your debugging or is there value in leaving it in?  If so maybe comment?

ditto for uses below.